lua/various-textobjs/linewise-textobjs.lua

```diff
@@ -141,7 +141,8 @@ end
 ---indentation textobj
 ---@param startBorder "inner"|"outer"
 ---@param endBorder "inner"|"outer"
-function M.indentation(startBorder, endBorder)
+function M.indentation(startBorder, endBorder, includeBlankLines)
+	includeBlankLines = includeBlankLines == nil and true or includeBlankLines
```

Comment:
to be consistent with the other variables, please use a string-enum variable, since that is more verbose.